eva-server/src/main/java/uk/ac/ebi/eva/server/repository/VariantEntityRepositoryImpl.java

```diff
@@ -53,18 +54,93 @@ public VariantEntityRepositoryImpl(MongoDbFactory mongoDbFactory, MappingMongoCo
         mongoTemplate = new MongoTemplate(mongoDbFactory, mappingMongoConverter);
     }
 
-    public List<VariantEntity> findByChrAndStartWithMarginAndEndWithMargin(String chr, int start, int end) {
+    private void queryConsequenceType(Query query, List<String> consequenceType) {
+        List<Integer> consequenceTypeConv = consequenceType.stream()
+                                                           .map(c -> Integer.parseInt(c.replaceAll("[^\\d.]", ""), 10))
+                                                           .collect(Collectors.toList());
+        query.addCriteria(Criteria.where("annot.ct.so").in(consequenceTypeConv));
+    }
+
+    private void queryMaf(Query query, String maf) {
```

Comment:
We need to test that resulting Query objects are well formed. It would be possible by converting the private methods into package protected.